cmd/juju/application/deploy.go

```diff
@@ -609,6 +612,7 @@ func (c *DeployCommand) SetFlags(f *gnuflag.FlagSet) {
 	f.Var(cmd.NewAppendStringsValue(&c.BundleOverlayFile), "overlay", "Bundles to overlay on the primary bundle, applied in order")
 	f.StringVar(&c.ConstraintsStr, "constraints", "", "Set application constraints")
 	f.StringVar(&c.Series, "series", "", "The series on which to deploy")
+	f.IntVar(&c.Revision, "revision", -1, "The revision to deploy")
```

Comment:
Do we additionally need to explain what '-1' means in this context?

Reply:
I don't think so, -1 as a revision is internal to juju and same as how charm.URL.Revision works.  Although, maybe a check for > -1 is a good idea.